packages/edit-site/src/components/page-pages/index.js

```diff
@@ -34,18 +35,13 @@ import {
 	OPERATOR_IS_NONE,
 } from '../../utils/constants';
 
-import {
-	trashPostAction,
-	usePermanentlyDeletePostAction,
-	useRestorePostAction,
-	postRevisionsAction,
-	viewPostAction,
-	useEditPostAction,
-} from '../actions';
 import AddNewPageModal from '../add-new-page';
 import Media from '../media';
 import { unlock } from '../../lock-unlock';
 
+const { postManagementActions } = unlock( editorPrivateApis );
+const { usePostActions } = postManagementActions;
```

Comment:
Do we expect future changes where consumers directly import the actions from `postManagementActions`? In this branch we only ever import `usePostActions`, so unless there are specific plans we could just export less:

```js
lock( privateApis, {
    // ...
    usePostActions
} );
```